src/datatypes/epgsql_codec_json.erl

```diff
@@ -17,18 +17,47 @@
 
 -define(JSONB_VERSION_1, 1).
 
-%% TODO: JSON encode/decode `fun Mod:Name/1` / `{Mod, Name}` as option.
-%% Shall not pass `fun(_) -> .. end`, because of hot code upgrade problems.
-init(_, _) -> [].
+%% JsonMod shall either be a module with encode/1 and decode/1, where:
+%%
+%% - encode/1 expects erlang-formatted JSON, e.g. a map or a proplist,
+%%   and returns a valid JSON binary string.
```

Comment:
I don't think we should insist on requirement to return binary - iolist might be also fine (however, I don't know any erlang JSON library that returns iolist)

Reply:
Sure!